session.go

```diff
@@ -811,3 +784,45 @@ func finishBootstrap(store kv.Storage) {
 		log.Fatalf("finish bootstrap err %v", err)
 	}
 }
+
+const loadCommonGlobalVarsSQL = "select * from mysql.global_variables where variable_name in ('" +
```

Comment:
There are two questions:
1. How do you write global vars to this table?
2. How about to set this table_name to a const string?